test/fixtures/doc-yaml/user-guide/secrets/secret-pod.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
 spec:
   containers:
     - name: test-container
-      image: gcr.io/kubernetes-e2e-test-images/mounttest:1.0
-      command: [ "/mt", "--file_content=/etc/secret-volume/data-1" ]
+      image: {{.MounttestImage}}
+      command: [ "/mounttest", "--file_content=/etc/secret-volume/data-1" ]
```

Comment:
is the /mt -> /mounttest change compatible? I don't see context for this

Reply:
It is, as you can see in the mounttest image's entrypoint: https://github.com/kubernetes/kubernetes/blob/master/test/images/mounttest/Dockerfile#L17

Reply:
somewhat confused still, I do see that, but where was `mt` before then? this change seems good but still possibly unrelated to the rest of the PR, did we recently change this?

Reply:
The changes are made long back part of - https://github.com/kubernetes/kubernetes/pull/44910/files#diff-c0ab7d1541310441bc515acd1db69ac2, I think we missed updating this file with right one.

Reply:
thanks @mkumatag ! I think we should maybe note this somewhere :-)

Reply:
Thanks @mkumatag for looking it up. :)